nemo/collections/nlp/losses/smoothed_cross_entropy.py

```diff
@@ -0,0 +1,119 @@
+# =============================================================================
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+from typing import List, Optional
+
+import torch
+from torch import nn
+
+from nemo.core.classes import Loss, typecheck
+from nemo.core.neural_types import LabelsType, LengthsType, LogitsType, LogprobsType, LossType, MaskType, NeuralType
+from nemo.utils.decorators import experimental
+
+__all__ = ['SmoothedCrossEntropyLoss']
+
+
+@experimental
+class SmoothedCrossEntropy(torch.nn.Module):
```

Comment:
why this class is needed? Is there something else which derives from it?
Also, why it inherits from torch.nn.Module and not torch.nn.modules.loss._Loss ?

Reply:
I have merged this into the other neuralmodule already in my PR.